### PR TITLE
Add asterisks to the required fields

### PR DESCRIPTION
**Pull request by cheemurakami**
### Describe your change in plain English.
Updated text field labels.

Screenshot 2023-09-02 at 3 03 59 PM

### Link to the issue

Fixes https://github.com/rubyforgood/pet-rescue/issues/33

### CODE REVIEW

File: app/views/organization_pets/_form.html.erb [hunk 1]
```diff
@@ -6,7 +6,7 @@
                             value: current_user.staff_account.organization_id %>
 
       <div class='form-group'>
-        <%= form.label :name, class: 'mt-3' %>
+        <%= form.label :name, 'Name*', class: 'mt-3' %>
```
Comment (edwinthinks) [inline app/views/organization_pets/_form.html.erb:9]: Could you check to see if adding `required: true` adds the asterisk for us? I normally see the asterisk added using that approach.

**Review by edwinthinks — CHANGES_REQUESTED** [review]
You should be able to use the `required:` field to add the asterisks. Let me know if this doesn't work.

File: app/views/organization_pets/_form.html.erb (hunk 1 above)
Comment (cheemurakami) [reply app/views/organization_pets/_form.html.erb:9]: Hi @edwinthinks ! I tried to use `required: true`, but it wasn't applying the asterisk. I followed [this file](https://github.com/rubyforgood/pet-rescue/blob/main/app/views/contacts/new.html.erb#L32-L35) 🤔

Comment (edwinthinks) [reply app/views/organization_pets/_form.html.erb]: Hmm I suspect it is because this is using form_with and not `form_for` could you see if that makes a difference?

**Review by edwinthinks — APPROVED** [review]
LGTM! Sorry for the delay @cheemurakami, we are reworking some major pieces of the codebase.
